### PR TITLE
Updates to Sources and help texts

### PR DESCRIPTION
**Pull request by jenniferward**

### CODE REVIEW

File: default/en/cat_printed.md [hunk 1]
```diff
@@ -101,9 +107,63 @@ Standard RISM rules for generic titles apply (see **Standardized title (240)**),
...
     - Title on source: Ander Theil \| Der Preussischen \| Fest-Lieder/ \| Von Ostern an biß Advent \| Mit 5/ 6/ 7/ 8. Stimmen. \| JOHANNIS ECCARDI MULHUSINI THURINGI, \| vnd \| JOHANNIS STOBAEI GRUDENTINI BORUSSI. \| Beyder Chur: vnd Fürstlicher Brandeb. Capell= \| meistern in Preussen. \| DISCANTVS. \| Gedruckt zu Königsberg durch Johann Reusnern An: 1644.  
-    - Standardized title: Preussische Festlieder  
-    - Additional title: Preussische Festlieder, Teil 2
+    - Standardized title: Preussische Festlieder, Teil 2
+
 
 ### Multiple copies in one institution
 
-If your institutions owns multiple copies (exemplars) of the same printed edition, create separate holdings records for each copy.
+If your institutions owns multiple copies (exemplars) of the same printed edition, create separate holdings records for each copy.
```
Comment (ahankinson) [inline default/en/cat_printed.md:115]:
```suggestion
If your institution owns multiple copies (exemplars) of the same printed edition, create separate holdings records for each copy.  
```

File: default/en/help_best_practices.md [hunk 2]
```diff
@@ -22,19 +20,28 @@ If you want to link to a website, do not enter the URL in a notes field. Rather,
...
+
+Catalogers should not remove information from the core record if it is correct, even if it is not relevant for local cataloging systems. Administrative information is stored in the records that can explain decisions made by the Editorial Center and the relationship to other records. Do not delete or edit data in the fields **Initial entry (775)**, **Local number (035)**, or **RISM series (510)**. Never delete a note that indicates that a record was merged with another record.
+
+If a record has already been revised by another working group (evident by the presence of a library's siglum and shelfmark in the field **Source of description note (588)**), use cataloger's judgement as to whether futher revision is necessary. If you make further changes, make it clear in the note what revisions you contributed.
+
+If your copy is incomlete, revisions might still be possible, but make it clear in the **Source of description note (588)** that your copy is incomplete.
```
Comment (ahankinson) [inline default/en/help_best_practices.md:45]:
```suggestion
If your copy is incomplete, revisions might still be possible, but make it clear in the **Source of description note (588)** that your copy is incomplete.
```

File: default/en/help_faqs.md [hunk 3]
```diff
@@ -35,6 +35,8 @@ This is a system edit. You might see a system edit if one of the indexed fields
 
 Muscat records that have the publication status "published" appear in the [RISM Catalog](https://opac.rism.info/), developed by the [Bavarian State Library](https://www.bsb-muenchen.de/) (Munich), and [RISM Online](https://rism.online/), developed by the [RISM Digital Center](https://rism.info/digital-center.html) (Bern, Switzerland). In addition, RISM records are available as open data and linked open data and may be reused in other projects.
 
+Outside of Muscat, published Muscat records are availble through RISM's [SRU downloader](https://github.com/rism-international/sru-downloader), the RISM Catalog's "Show MARCXML" link on any record, the RISM Catalog's monthly data dump, and RISM Online's API.
+
 #### 8. When do Muscat records appear in the public catalogs?
 
 Muscat records are sent for publication in the [RISM Catalog](https://opac.rism.info/) once a month, at around the 19th of each month. Records are then visible a few days later.
```
Comment (ahankinson) [inline default/en/help_faqs.md:42]:
```suggestion
Muscat records are sent for publication in the [RISM Catalog](https://opac.rism.info/) once a month, at around the 19th of each month. Records are then visible a few days later.  [RISM Online](https://rism.online) is updated daily, so new records and changes will appear there on the following day.
```

File: default/en/institution_856.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ Enter information (including URLs) about finding aids, catalogs, inventories, et
 
 #### Online finding aids (856 $u)
 
-Enter the URL of the finding aid.
+Enter the URL of the finding aid. Only enter one URL per field. All URLs must begin with http.
```
Comment (ahankinson) [inline default/en/institution_856.md:10]:
```suggestion
Enter the URL of the finding aid. Only enter one URL per field. All URLs must begin with `http` or `https`.
```

File: default/en/source_260.md [hunk 5]
```diff
@@ -96,6 +98,8 @@ For approximate periods of time, enter a range of years based on the following s
 - 18.sc (18th century)
 - 18/19 (18th or 19th century)
 
+Furthermore, previous cataloging programs had two fields for dating. One was for numbers only and the second was a text field where numbers plus abbreviations such as _+, ca., ante, post,_ and _18.sc_ or text such as _Easter 1805_ or _?1745?_ could be entered. Some legacy records, therefore, display two date indications, such as _1806 (1806 ca.)_. Dates written in the format XX.XX.XXXX may be interpreted as DD.MM.YYYY.
```
Comment (ahankinson) [inline default/en/source_260.md:101]:
```suggestion
Furthermore, previous cataloging programs had two fields for dating. One was for numbers only and the second was a text field where numbers plus abbreviations such as _+, ca., ante, post,_ and _18.sc_ or text such as _Easter 1805_ or _?1745?_ could be entered. Some legacy records, therefore, display two date indications, such as _1806 (1806 ca.)_. Dates written in the format XX.XX.XXXX are interpreted as DD.MM.YYYY.
```

**Review by ahankinson — CHANGES_REQUESTED** [review]

**Review by jenniferward — COMMENTED** [review]
Thank you! I deleted the questionable sentence and approved the other changes.

File: default/en/source_031.md [hunk 6]
```diff
@@ -128,7 +132,7 @@ be added in square brackets.
 
 If there is no key signature, leave the field blank.  
 
-If there are multiple key signatures (such as if a movement changes frequently from major to minor and back again), enter the key signature of the incipit in question add an explanation in the field **General note (031 $q).**
+If there are multiple key signatures (such as if a movement changes frequently from major to minor and back again), enter the key signature of the incipit in question add an explanation in the field **General note (031 $q)**.
```
[the remark this answers is not in the document]
Comment (jenniferward) [reply default/en/source_031.md]: I think I changed it (i.e., I deleted it).

**Review by ahankinson — APPROVED** [review]

**Pull request merged**